dev-util/flatpak-builder/flatpak-builder-0.9.11.ebuild

```diff
@@ -0,0 +1,37 @@
+# Copyright 1999-2007 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
+
+SRC_URI="https://github.com/flatpak/${PN}/releases/download/${PV}/${P}.tar.xz"
+DESCRIPTION="Tool for building flatpaks from sources"
+HOMEPAGE="http://flatpak.org/"
+
+LICENSE="LGPL-2.1"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="doc"
+
+RDEPEND="
+	>=sys-apps/flatpak-${PV}
+	>=sys-fs/libostree-2017.10
+	>=net-libs/libsoup-2.4
+	>=dev-libs/elfutils-0.8.12
+	>=dev-libs/glib-2.44:2
+	dev-libs/json-glib
+"
+DEPEND="${RDEPEND}
+	>=sys-devel/gettext-0.18.2
+	virtual/pkgconfig
+	>=dev-libs/gobject-introspection-1.40
```

Comment:
gobject-introspection was [dropped](https://github.com/flatpak/flatpak-builder/commit/c011526e875d4429fbe266de7bc5b9040c05a5f2#diff-67e997bcfdac55191033d57a16d1408a)

Reply:
Ah, missed that, thanks.

Reply:
And when updating the ebuild you also need to re-digest.

Reply:
```
$ sed -e '/0.9.11/d' -i dev-util/flatpak-builder/Manifest
$ ebuild dev-util/flatpak-builder/flatpak-builder-0.9.11.ebuild digest
Appending /home/ace/Projects/Gentoo/flatpak-overlay to PORTDIR_OVERLAY...
>>> Creating Manifest for /home/ace/Projects/Gentoo/flatpak-overlay/dev-util/flatpak-builder
$ git status
On branch bump-flatpak
Your branch is up-to-date with 'origin/bump-flatpak'.

nothing to commit, working tree clean
```

Reply:
oh wait, the repo is on thin-manifest, so it isn't needed for the ebuilds.

One more thing however is that I usually just delete the old ebuild for the SLOT.

Reply:
You seem to still have the 0.8.7 flatpak ebuild in place though?

I assume you mean for the minor version then?

Reply:
yeah.. i shouldn't have said slot. I meant stable/unstable. I want to keep a stable version around for ppl who prefer it.

Reply:
Could you squash the commits?

Reply:
And separate them out per package? (1 commit for builder, 1 commit for general)